packages/event-handler/src/rest/converters.ts

```diff
@@ -0,0 +1,48 @@
+import type { APIGatewayProxyEvent } from 'aws-lambda';
+
+function createBody(body: string | null, isBase64Encoded: boolean) {
```

Comment:
We generally try to use arrow functions across the project, except for cases where we need the scope.